src/Panel/Panel/Panel.jsx

```diff
@@ -306,14 +349,12 @@ export class Panel extends React.Component {
     return (
       <Rnd
         className={rndClassName}
-        ref={(panel) => { this.panel = panel; }}
+        ref={(rnd) => { this.rnd = rnd; }}
         default={{
           x: isNumber(rndOpts.x) ? rndOpts.x : (window.innerWidth / 2) - 160,
           y: isNumber(rndOpts.y) ? rndOpts.y : (window.innerHeight / 2) - 100,
```

Comment:
Not related to this PR, but just curious where are these numbers (160 & 100) are from.

Reply:
I don't know.